blaze/expr/strings.py

```diff
@@ -74,6 +84,54 @@ def schema(self):
         return self._child.schema
 
 
+class StrCat(ElemWise):
+    """
+    Concatenate two string columns together with optional 'sep' argument.
+
+    >>> import pandas as pd
+    >>> from blaze import symbol, compute, dshape
+
+    >>> ds = dshape('3 * {name: string[10], comment: string[25], num: int32}')
+    >>> s = symbol('s', dshape=ds)
+    >>> data = [('al', 'good', 0), ('suri', 'not good', 1), ('jinka', 'ok', 2)]
+    >>> df = pd.DataFrame(data, columns=['name', 'comment', 'num'])
+
+    >>> compute(s.name.str_cat(s.comment, sep=' -- '), df)
+    0          al -- good
+    1    suri -- not good
+    2         jinka -- ok
+    Name: name, dtype: object
+
+    """
+    __slots__ = '_hash', 'lhs', 'rhs', 'sep'
+    __inputs__ = 'lhs', 'rhs'
+
+    def _dshape(self):
+        '''
+        since pandas supports concat for string columns, do the same for blaze
+        '''
+        shape, schema = self.lhs.dshape.shape, DataShape(String())
+        return DataShape(*(shape + (schema,)))
+
+
+@copydoc(StrCat)
+def str_cat(expr, to_concat, sep=None):
```

Comment:
To emphasize the symmetry and to match the corresponding `StrCat` instance attributes, can you rename `expr` to `lhs` and `to_concat` to `rhs`?